Partial-CI-Tests-Structure.md

```diff
@@ -0,0 +1,17 @@
+## Introduction
+
+If your PR fails due to the `check_ci_test_suites_to_run` workflow, please look at the error log and depending on the error, do the following:
+
+* If you see an error about the root files config, then see the [root files config section](#root-files-config) below.
+
+## Overview of Partial CI Tests Structure
+
+For non-python files, we run partial ci tests depending on the tests that a specific file affects. The process of determining which tests to run goes like this:
+
+1. Generate a root files mapping which maps files to their root files.
```

Comment:
Here and below: explain where in the codebase this is stored -- perhaps link to the specific files etc. That makes this much more concrete and developers can examine the code. Currently it's a bit vague.

Reply:
Done.

Reply:
Done